pkgs/by-name/xp/xpra-html5/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3,
+  uglify-js,
+}:
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenvNoCC.mkDerivation rec {
```
Also I think `let ... in` is preferred to `rec`.

Reply:
It doesn't really matter